modules/network-observability-operator-install.adoc

```diff
@@ -13,19 +13,21 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Procedure
 
-.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
-//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O. 
-.. Choose  *Network Observability Operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
-.. Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* link.
-. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O.
+. Choose  *Network Observability Operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
+. Select the checkbox `Enable Operator recommended cluster monitoring on this Namespace`.
+. Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* link.
+.. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
 +
 * *spec.agent.ebpf.Sampling* : Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the `FlowCollector` API reference, under spec.agent.ebpf. 
 * *spec.deploymentModel*: If you are using Kafka, verify Kafka is selected.
 * *loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", should match the name of your LokiStack.
 * *loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`.
```

Comment:
even statusUrl needs to say the same thing as loki.Url. Basically this statement needs to be added `The first part of the URL, "loki", should match the name of your LokiStack.`

Reply:
This is still needed according to me